ubuntudesign/www.ubuntu.com
# use a single form for all iot contact us pages

## Done

* created a new view and single form for all the iot contact us forms to use

## QA

1. go to all the pages in the iot section and see that the contact us page changes it heading, intro text and form_id based on where you are

## Issue / Card

fixes #1089
